### DISCUSSION TITLE
ModuleNotFoundError: No module named 'h2o_wave'

### DISCUSSION THREAD

**Discussion by Everything-is-one** [h2oai/wave#2326] [Q&A]
In accordance with the guide document (https://wave.h2o.ai/docs/installation) installed h2o - run after wave "wave run error.

Error loading ASGI app. Could not import module "button".

Traceback (most recent call last):
  File "/root/miniconda3/envs/h2o/bin/wave", line 6, in <module>
    from h2o_wave.cli import main
ModuleNotFoundError: No module named 'h2o_wave'
![Snipaste_2024-04-29_12-57-43](https://github.com/h2oai/wave/assets/92569428/9eafd53a-c795-49ed-9010-883f9e283eb5)
![Snipaste_2024-04-29_12-14-46](https://github.com/h2oai/wave/assets/92569428/54f232f6-b666-4f83-a49e-fd4893b1fa28)
![Snipaste_2024-04-29_14-11-54](https://github.com/h2oai/wave/assets/92569428/de9c2032-87aa-466c-9e44-8544a6019f5a)

**Comment by mturoci** [comment]
Yea, this is not working on conda at the moment. You will need to switch to pip instead.
